spec/controllers/ems_common_controller_spec.rb

```diff
@@ -169,7 +169,24 @@
 end
 
 describe EmsContainerController do
+  before :each do
+    allow(controller).to receive(:get_hostname_from_routes).and_return("nshneor-centos7-1.eng.lab.tlv.redhat.com")
```

Comment:
The method `get_hostname_from_routes` is now used by all the methods that create or refresh `ExtManagementSystem`, thus we need to stub it before all the tests.
I also stubbed it in the file `refresher_rhos_juno_spec.rb` for the same reason.